shotover-proxy/tests/test-configs/shotover-config/config_metrics_disabled.yaml

```diff
@@ -1 +1 @@
-main_log_level: "info, shotover::connection_span=debug"
+main_log_level: "info, shotover::connection_span=debug"
```

Comment:
I think you've deleted one too many lines here 😅 

Each line should end with exactly one newline, and this single line file has no newlines at all.

Github shows that the newline was removed via the little icon:
<img width="34" alt="image" src="https://github.com/user-attachments/assets/0e4a621d-61b3-4d15-9530-9ea43e80e3db" />

I'm not usually fussed about this kind of formatting for new files, but as a reviewer I dont want existing files that have no changes showing up in the diff.

If your confused, you should be able to restore the original file by running: `git checkout main shotover-proxy/tests/test-configs/shotover-config/config_metrics_disabled.yaml` and then just commit that.